src/components/SearchInput/SearchInput.stories.tsx

```diff
@@ -9,5 +9,5 @@ export default {
 type Story = StoryObj<typeof SearchInput>;
 
 export const Default: Story = {
-  args: {},
+  args: { placeholder: 'Search', inputType: 'input', onChange: () => {} },
```

Comment:
onChange is empty function is not correct, please read this https://storybook.js.org/docs/essentials/actions

Reply:
Fixed